angular-ngrx-scss/src/app/home/home.component.ts

```diff
@@ -0,0 +1,12 @@
+// import { Component, OnInit } from '@angular/core';
```

Comment:
Why is this commented out entirely? 🤔

Reply:
For now, because it was failing the linter from everything being empty. The next task is what will actually start to fill this out with data. It's a temporary thing - I need the component in place to ensure the auth routing works, but it doesn't need to actually do anything yet.

Reply:
Didn't realize this was causing other errors in the app, so fixed this to remove the parts it didn't need. It's now not commented out, and still doesn't have any functionality.